articles/aks/troubleshooting.md

```diff
@@ -28,6 +28,37 @@ You need to [request cores](https://docs.microsoft.com/azure/azure-supportabilit
 The maximum pods-per-node setting is 30 by default if you deploy an AKS cluster in the Azure portal.
 The maximum pods-per-node setting is 110 by default if you deploy an AKS cluster in the Azure CLI. (Make sure you're using the latest version of the Azure CLI). This default setting can be changed by using the `–-max-pods` flag in the `az aks create` command.
 
+## Is it safe if the IP address ranges overlap with one another or with on-premise networks?
+
+Overlapping subnets and IP ranges could cost several days of lost productivity if issues come up because of deviation from recommended practices.
+ 
+This could help shed some light on some of the issues one could encounter as a result of such overlaps: 
+ 
+In advanced networking scenarios, the IP address plan for an AKS cluster consists of the following partitions:
```

Comment:
I'd avoid terms of "advanced and basic" networking, we should talk on the lines of the 4 combinations:
Managed subnet:
 - Kubenet
 - CNI

BYO Subnet
 - kubenet
 - CNI